cypress/integration/tests/developer/applicationinventory/analysis/source_analysis.test.ts

```diff
@@ -113,6 +125,7 @@ describe("Source Analysis", { tags: "@tier1" }, () => {
         application.verifyAnalysisStatus("Completed");
         application.openReport();
         application.validateStoryPoints();
+        application.downloadReport("HTML");
```

Comment:
This file does not need HTML and CSV as we have covered it in the download report test.